src/Quartz.Extensions.DependencyInjection/Simpl/JobActivatorCache.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Concurrent;
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Quartz.Simpl
+{
+    public class JobActivatorCache
```

Comment:
Can we make this internal? I'd rather hide everything we can and only open when request with good reasoning comes in. Makes it easier to move the cheese.

Reply:
It's internal now.